defaults.h

```diff
@@ -51,4 +51,7 @@
 
 #endif /* !HAVE_DATACENTER */
 
+#define DFLT_VRF_NAME	"Default-IP-Routing-Table"
```

Comment:
Why not use VRF_DEFAULT_NAME? - having two basically identical defines doesn't make sense to me...

Reply:
actually, this seems to be my speciality :-).
I made 2 features in one:
a- global defined name
b- dynamic global vrf name with netns

because of b, I mapped VRF_DEFAULT_NAME to vrf_get_default_name(void)

Reply:
I will care of the fact that VRF_DEFAULT_NAME will not be used in other places.
if netns defines a new name, then this will be an other define., but not VRF_DEFAULT_NAME.
in order to avoid confusion